.gitignore

```diff
@@ -2,8 +2,8 @@
 /dist/
 __pycache__/
 .coverage
-.DS_Store
```

Comment:
Is this intentional?

Reply:
Yes! It was duplicated, the other one is still there, and I checked that it still works (my machine doesn't show diffs for `.DS_Store` now, but it does if I remove the other remaining `.DS_Store` from `.gitignore).